composer.json

```
@@ -28,9 +28,9 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.3 || ~8.0",
```

Be careful here:

> The ~ operator is best explained by example: `~1.2` is equivalent to `>=1.2 <2.0.0`, while `~1.2.3` is equivalent to `>=1.2.3 <1.3.0`.

https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

```suggestion
        "php": "^7.3 || ~8.0.0",
```